open-metadata-implementation/integration-services/search-integrator/search-integrator-api/src/main/java/org/odpi/openmetadata/integrationservices/search/connector/SearchIntegratorConnector.java

```diff
@@ -0,0 +1,65 @@
+/* SPDX-License-Identifier: Apache 2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
+
+package org.odpi.openmetadata.integrationservices.search.connector;
+
+import org.odpi.openmetadata.frameworks.connectors.ffdc.ConnectorCheckedException;
+import org.odpi.openmetadata.governanceservers.integrationdaemonservices.connectors.IntegrationConnectorBase;
+import org.odpi.openmetadata.integrationservices.search.ffdc.SearchIntegratorAuditCode;
+import org.odpi.openmetadata.integrationservices.search.ffdc.SearchIntegratorErrorCode;
+
+/**
+ * LineageIntegratorConnector is the base class for an integration connector that is managed by the
```

Comment:
Should LineageIntegratorAPI be SearchIntegratorAPI

Reply:
updated